src/database/migrations/20221012211534_create_diving_tank_set_tables.ts

```diff
@@ -0,0 +1,36 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('diving_cylinder', (table) => {
+    table.uuid('id').primary();
+    table.integer('volume').notNullable();
+    table.integer('pressure').notNullable();
+    table.string('material', 32).notNullable();
+    table.string('serial_number', 64).notNullable();
+    table.date('inspection').notNullable();
+    table.timestamps(true, true);
+  });
+
+  await knex.schema.createTable('tank_set', (table) => {
+    table.uuid('id').primary();
+    table.uuid('owner').references('id').inTable('user').notNullable();
+    table.string('name', 255).notNullable();
+    table.timestamps(true, true);
+    table.unique(['owner', 'name']);
+  });
+
+  await knex.schema.createTable('tank_set_diving_cylinder', (table) => {
+    table
+      .uuid('cylinder')
+      .references('id')
+      .inTable('diving_cylinder')
+      .primary();
+    table.uuid('tank_set').references('id').inTable('tank_set').primary();
+  });
+}
+
+export async function down(knex: Knex): Promise<void> {
+  await knex.schema.dropTable('tank_set_diving_cylinder');
+  await knex.schema.dropTable('tank_set');
+  await knex.schema.dropTable('diving_cylinder');
```

Comment:
```suggestion
  await knex.schema.dropTableIfExists('tank_set_diving_cylinder');
  await knex.schema.dropTableIfExists('tank_set');
  await knex.schema.dropTableIfExists('diving_cylinder');
```

Olisko tää hyvä olla?

Reply:
Varmaankin

Reply:
Devaustahan tuo vain helpottaa. Ei mitään merkitystä tuotannossa mielestäni